sdk/src/utils.ts

```diff
@@ -109,6 +109,41 @@ export function createTx(
 }
 
 export async function sendTx(
+  tx: SubmittableExtrinsic,
+  waitForFinalization: boolean = true,
+  block: boolean = true
+): Promise<EventTxResult<any> | TxResult> {
+  return new Promise(async (resolve, reject) => {
```

Comment:
same part logic as in `sendTx` and `signAndSendTx`. Use single point of definition